pkg/kubecost/allocationprops.go

```diff
@@ -93,6 +96,7 @@ func ParseProperty(text string) (string, error) {
 type AllocationProperties struct {
 	Cluster        string                `json:"cluster,omitempty"`
 	Node           string                `json:"node,omitempty"`
+	NodeType       string                `json:"nodeType,omitempty"`
```

Comment:
Reading this kind of gets me excited, actually--I think there could be some interesting Allocation-related analysis done with this field.